frequencyChar.js

```diff
@@ -0,0 +1,20 @@
+function itemFrequency(array) {
+    let characterFrequency = {};
+    let totalCount = 0
+  
+    for (let item of array) {
+        totalCount+= 1
+      if (!characterFrequency[item]) {
+        // if not in the dictionary
+        characterFrequency[item] = 1;
+      } else {
+        characterFrequency[item] += 1;
+      }
+    } for (let keys of Object.keys(characterFrequency)) {
+       characterFrequency[keys]= (characterFrequency[keys]/totalCount)*100 +' %';
```

Comment:
If you store the frequency like this, you'll have a hard time creating a useful histogram display.

Generally, we want to keep the concern of displaying information separate from the concern of storing / encoding the information. We often want to use the same information in many ways.

For example, in this project, you'll want to use it both to display a percentage and to scale the width of each histogram part. Encoding it as a string will make that difficult.

See https://en.wikipedia.org/wiki/Separation_of_concerns